.circleci/README.md

```diff
@@ -0,0 +1,31 @@
+# CircleCI
+
+CircleCI is used to run unit tests on Unix env.
+
+## Upgrading Golang version
+
+/!\ Disclaimer: the datadog-agent-runner-circle should never be used for anything else than CircleCI tests /!\
+
+Change the Golang version in this file `images/runner/Dockerfile#L13`.
```

Comment:
I'd only link the file, not the line (there's no guarantee the go version definition will remain on line 13)

Reply:
I don't know for the image, I could not found where it's used